docs/source/quickstart.mdx

```diff
@@ -1,18 +1,18 @@
 ---
-title: Apollo Router Quickstart
-subtitle: Run the Apollo Router with Apollo-hosted subgraphs
-description: This quickstart tutorial walks you through installing the Apollo Router and running it in front of some Apollo-hosted example subgraphs.
+title: GraphOS Router Quickstart
```

Comment:
Available in both Router Core and GraphOS Router (for all occurrences on this page)